src/qt/overviewpage.cpp

```diff
@@ -107,9 +106,9 @@ class TxViewDelegate : public QAbstractItemDelegate
 
         painter->setPen(option.palette.color(QPalette::Text));
         QRect date_bounding_rect;
-        painter->drawText(amountRect, Qt::AlignLeft | Qt::AlignVCenter, GUIUtil::dateTimeStr(date), &date_bounding_rect);
+        painter->drawText(amountRect, Qt::AlignLeft | Qt::AlignVCenter, GUIUtil::dateTimeStr(date) + QLatin1String("  "), &date_bounding_rect);
```

Comment:
Is this assuming spaces have a certain width?

Reply:
Do you prefer non-breaking space?

Reply:
Offset the bounding rect?

Spaces are never a well-defined width... except zero-width :)

Reply:
Offset in fixed number of pixels? Is it a good approach with potentially scalable fonts?

Reply:
Measure the icon width.

Scalable fonts is exactly why spaces won't work reliably. ;)

Reply:
> Is this assuming spaces have a certain width?

Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/365#issuecomment-869523331).